homeassistant/components/tessie/sensor.py

```diff
@@ -81,6 +83,14 @@ class TessieSensorEntityDescription(SensorEntityDescription):
         device_class=SensorDeviceClass.SPEED,
         entity_category=EntityCategory.DIAGNOSTIC,
     ),
+    TessieSensorEntityDescription(
+        key="charge_state_time_to_full_charge",
+        device_class=SensorDeviceClass.TIMESTAMP,
+        entity_category=EntityCategory.DIAGNOSTIC,
+        value_fn=lambda x: (dt_util.now() + timedelta(hours=x))
+        if isinstance(x, (int, float)) and x > 0
+        else None,
```

Comment:
We typically try to avoid multiline ternary lambdas, as they're hard to read. Can we move this into a small function?

Reply:
Done.